SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/ReferenceCountedLockingPolicy/ReferenceCountedLockingPolicyTest.cpp

```diff
@@ -187,21 +211,12 @@ bool FReleaseLock::Update()
 	return true;
 }
 
-DEFINE_LATENT_AUTOMATION_COMMAND_FIVE_PARAMETER(FTestIsLocked, FAutomationTestBase*, Test, TSharedPtr<TestData>, Data, FName, Handle, bool, bIsLockedExpected, int32, LockTokenCountExpected);
+DEFINE_LATENT_AUTOMATION_COMMAND_FOUR_PARAMETER(FTestIsLocked, FAutomationTestBase*, Test, TSharedPtr<TestData>, Data, FName, Handle, bool, bIsLockedExpected);
 bool FTestIsLocked::Update()
 {
 	const AActor* Actor = Data->TestActors[Handle];
 	const bool bIsLocked = Data->LockingPolicy->IsLocked(Actor);
-
-	TSet<ActorLockToken> LockTokens;
-	int32 LockTokenCount = 0;
-	if (bIsLocked)
-	{
-		LockTokenCount = Data->TestActorToLockingTokenAndDebugStrings[Actor].Num();
-	}
-	
 	Test->TestEqual(FString::Printf(TEXT("Is locked. Actual: %d. Expected: %d"), bIsLocked, bIsLockedExpected), bIsLocked, bIsLockedExpected);
-	Test->TestEqual(FString::Printf(TEXT("Lock count. Actual: %d. Expected: %d"), LockTokenCount, LockTokenCountExpected), LockTokenCount, LockTokenCountExpected);
```

Comment:
This method was changed because it was basically testing the test, rather than testing the actual class